### PR TITLE
New Chrome Cheat Sheet for Mac

### PR DESCRIPTION
**Pull request by vikashvverma**
IA Page: https://duck.co/ia/view/chrome_mac_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/chrome_mac.json [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    "description": "Google Chrome Keyboard shorcuts (Mac)", 
+    "metadata": { 
+        "sourceName": "Chrome Help",
+        "sourceUrl": "https://support.google.com/chrome/answer/157179?hl=en"
+    },
+    "aliases": [
+        "google chrome mac",
+        "chrome mac keyboard shortcuts",
+        "google chrome mac keyboard shortcuts",
+        "chrome mac shortcuts"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/chrome_mac.json]: Lose the last three aliases.

File: share/goodie/cheat_sheets/json/chrome_mac.json [hunk 2]
```diff
@@ -0,0 +1,157 @@
+{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/chrome_mac.json]: @vikashvverma please change the filename to `chrome-mac.json`

**Comment by moollaza** [comment]
@vikashvverma Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

File: share/goodie/cheat_sheets/json/chrome-mac.json [hunk 3]
```diff
@@ -0,0 +1,157 @@
+{
+    "id": "chrome_mac_cheat_sheet", 
+    "name": "Chrome Mac",
+    "description": "Google Chrome Keyboard shorcuts (Mac)",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/chrome-mac.json]: @vikashvverma  Lets use small letter k for keyboard

**Comment by vikashvverma** [comment]
@moollaza This can be merged now!

**Comment by GuiltyDolphin** [comment]
@duckduckgo/duckduckhack-contributors Does anyone use this and have any feedback?

**Comment by ethanchewy** [comment]
Does anyone have a link to an example query in the beta duckduckgo website?

daxtheduck seems to have issues with deploying it...

**Comment by mintsoft** [comment]
@ethanchewy it's not deployed; looks like there was a problem

**Comment by alohaas** [comment]
let me take a look...

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=chrome mac cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/chrome_mac_cheat_sheet_mobile.png?H1bZLrUesK)

**Comment by alohaas** [comment]
This looks good to me 👍  Merging.

**Comment by daxtheduck** [comment]
@vikashvverma this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**